<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ResetPassword</class>
 <widget class="QDialog" name="ResetPassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1010</width>
    <height>726</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>211</width>
     <height>261</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="title">
    <string>Reset Password</string>
   </property>
   <widget class="QLineEdit" name="newPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="ConfirmNewPasswordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>154</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm New Password</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ConfirmNewPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>170</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="oldPasswordLineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>137</width>
      <height>36</height>
     </rect>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
   </widget>
   <widget class="QLabel" name="newPasswordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>90</y>
      <width>98</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>New Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="oldPasswordLabel">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Old Password</string>
    </property>
   </widget>
   <widget class="QPushButton" name="confirmButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>210</y>
      <width>88</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Confirm </string>
    </property>
   </widget>
   <widget class="QPushButton" name="cancelButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>210</y>
      <width>88</width>
      <height>36</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
